examples/quads.c

```diff
@@ -0,0 +1,216 @@
+#define _POSIX_C_SOURCE 200112L
+#define _XOPEN_SOURCE 600
```

Comment:
Specify one of these two, but not both. `_XOPEN_SOURCE` automatically enables a specific version of `_POSIX_C_SOURCE`, specifying both with a mismatch confuses some libc implementations.